documentation/book/proc-enabling-tracing-in-connect-mirror-maker-resources.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// assembly-setting-up-tracing-mirror-maker-connect.adoc
+
+[id='proc-enabling-tracing-in-connect-mirror-maker-resources-{context}']
+= Enabling tracing in the KafkaMirrorMaker, KafkaConnect, and KafkaConnectS2I custom resources
+
+Update the configuration of the `KafkaMirrorMaker`, `KafkaConnect`, and `KafkaConnectS2I` custom resources to specify and configure a Jaeger tracer service. When a tracing-enabled resource is updated in your Kubernetes cluster, two actions are triggered:
+
+* The Interceptors for the consumers and producers that are integrated into Mirror Maker or Kafka Connect are updated.
+
+* The tracing agent initializes a Jaeger tracer based on the tracing configuration in the resource.
+
+.Procedure
+
+Perform these steps for each `KafkaConnect`, `KafkaConnectS2I`, and `KafkaMirrorMaker` resource.
+
+. In the `spec.template` property, configure the Jaeger tracer service. Use the xref:ref-tracing-environment-variables-{context}[tracing environment variables] as template configuration properties. For example:
++
+.Jaeger tracer configuration for Kafka Connect
+[source,yaml,subs=attributes+]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+spec:
+  #...
+  template:
+    connectContainer:
+      env:
+        - name: JAEGER_SERVICE_NAME
+          value: my-jaeger-service
+#...
```

Comment:
It would be good to include a more realistic example here if we can.

Reply:
https://github.com/adam-cattermole/strimzi-lab/blob/add-taxi-example/taxi-example/doc/modules/tracing-guide.adoc